docs/user-guide/examples/advanced_examples/advanced_orchestration.md

```diff
@@ -7,4 +12,342 @@
 
 ```
 
-text
+This tutorial will walk through how to set up a HELICS Monte Carlo simulation using two techniques: (1) in series on a single machine, and (2) in parallel on an HPC cluster using Merlin. We assume that you have already completed the
+[**orchestration tutorial with Merlin**](../../advanced_topics/orchestration.md) and have some
+familiarity with how Merlin works.
+
+- [Where is the code?](#where-is-the-code)
+- [What is this Co-simulation doing?](#what-is-this-co-simulation-doing)
+- [Probabilistic Uncertainty Estimation](#probabilistic-uncertainty-estimation)
+- [Execution and Results](#execution-and-results)
+  - [Manual Orchestration Execution](#manual-orchestration-execution)
+  - [Merlin Orchestration Execution](#merlin-orchestration-execution)
+
+## Where is the code?
+
+Code for the Monte Carlo simulation and the
+full Merlin spec and be found in the [HELICS Examples Repo](https://github.com/GMLC-TDC/HELICS-Examples/tree/master/user_guide_examples/advanced/advanced_orchestration). If you have issues navigating to the examples, visit the HELICS Gitter page or the user forum on GitHub.
+
+[![](../../../img/advanced_orchestration_github.png)](https://github.com/GMLC-TDC/HELICS-Examples/tree/master/user_guide_examples/advanced/advanced_orchestration)
+
+The necessary files are:
+
+- Python program for Battery federate
+- Python program for Charger federate
+- Python program to generate `helics_cli` JSON files and execute
+
+## What is this co-simulation doing?
+
+This example walks through how to set up a probabilistic model with Monte Carlo simulations. This Monte Carlo co-simulation is built from a simple two federate example, based on the [Endpoint Federates Example](../fundamental_examples/fundamental_endpoints.md). In this example, there is a Charger federate which publishes voltage and a Battery federate which publishes current.
+
+All of the HELICS configurations are the same as in the Endpoint example. The internal logic of the federates has been changed for this implementation. The Charger federate assumes the role of _deciding_ if the Battery should continue to charge. The Battery sends a message of its current state of charge (soc, a number between 0 and 1). If the soc is less than 0.9, the Battery is instructed to continue charging, otherwise, it is instructed to cease charging. The Battery federate has all the logic internal for adding energy and selecting a new "battery" (charging rate) if the soc is deemed sufficient. Energy is added to the "battery" according to the previous time interval and the charge rate of the battery. In this way, the only stochastic component to the system is the **selected charge rate**. For example, the Endpoint Example allowed the Battery federate to randomly select batteries of different sizes, and the Charger to select charge rates from a list of options. In this implementation, the battery size (capacity in kWh) is constant.
```

Comment:
Reorder `soc` and expanded version, remove the value range from the parenthesis to make it the abbreviation meaning clearer.

```suggestion
All of the HELICS configurations are the same as in the Endpoint example. The internal logic of the federates has been changed for this implementation. The Charger federate assumes the role of _deciding_ if the Battery should continue to charge. The Battery sends a message of its current soc (state of charge), a number between 0 and 1. If the soc is less than 0.9, the Battery is instructed to continue charging, otherwise, it is instructed to cease charging. The Battery federate has all the logic internal for adding energy and selecting a new "battery" (charging rate) if the soc is deemed sufficient. Energy is added to the "battery" according to the previous time interval and the charge rate of the battery. In this way, the only stochastic component to the system is the **selected charge rate**. For example, the Endpoint Example allowed the Battery federate to randomly select batteries of different sizes, and the Charger to select charge rates from a list of options. In this implementation, the battery size (capacity in kWh) is constant.
```